README.md

```diff
@@ -175,3 +175,9 @@ services:
 ```
 
 In the above examples the mailhog interface will be accessible on port 8025.
+
+
+## [Wait application](https://github.com/ufoscout/docker-compose-wait)
+
+Allows for waiting on containers, that containers created from these images
+depends upon.
```

Comment:
"It can wait"... funny.

Since we are just pulling the newest version I was under the impression that we might just link to the newest docs. But then again.. We are only on the newest version as of when the image have been build last. Of course.. How to handle this?

Reply:
Could be just run `wait` by default and if no environment variables have been set then it is effectively a noop? This would lower the barrier of entry.

Reply:
Good input @kasperg! I have not tried doing that. I'll try and add it.